src/components/DashboardLayout/SelectedInstanceInfo.tsx

```diff
@@ -41,7 +43,23 @@ export const SelectedInstanceInfo = ({
 
   const uuid = selectedInstance?.uuid;
   if (!selectedInstance || !uuid) {
-    return <div></div>;
+    return (
+      <div className="mx-1 px-1 text-gray-faded/30">
+        <div className="text-small font-bold leading-snug text-gray-faded/30">
+          SELECTED INSTANCE
+        </div>
+        <div className="mt-[0.5rem] flex h-[17.61rem] justify-center rounded-md border border-dashed text-center text-gray-faded/30">
```

Comment:
Try to avoid arbitrary decimal `rem`, it causes weird subpixel layout shifting.
Notice how the icons are aligned but the text is mis-aligned by 1px (or less!)

If the rem number has like, crazy small decimals, maybe just hard code both the placeholder & the normal layout to some nice round number of rem?

![image](https://user-images.githubusercontent.com/44907675/224599480-e2d5137b-6e28-4898-8704-afc909b6057d.png)